src/constants/lists.ts

```diff
@@ -22,6 +22,10 @@ const BAOSWAP_LIST = 'https://raw.githubusercontent.com/baofinance/tokenlists/ma
 const QUICKSWAP_LIST = 'https://unpkg.com/quickswap-default-token-list@1.2.29/build/quickswap-default.tokenlist.json'
 const DFYN_LIST = 'https://raw.githubusercontent.com/dfyn/new-host/main/list-token.tokenlist.json'
 const OPTIMISM_LIST = 'https://static.optimism.io/optimism.tokenlist.json'
+const PANCAKESWAP_LIST =
+  'https://raw.githubusercontent.com/pancakeswap/pancake-toolkit/master/packages/token-lists/lists/pancakeswap-default.json'
+const SUSHISWAP_LIST =
```

Comment:
why we also added sushi swap?

Reply:
I added sushiswap as a usable exchange in the BSC chain. As a consequence I thought I could also add its list.
Reviewing this, I am facing with a format problem. This list is not similar to other lists... as in, it doesn't include metadata information and the tokens aren't under a "tokens" key.
I might remove it for now.